--- asyncio_helper.py	2023-01-30 23:41:29.118677694 +0000
+++ telegram/lib/python3.10/site-packages/telebot/asyncio_helper.py	2023-01-30 23:43:03.662310804 +0000
@@ -244,7 +244,7 @@
     if timeout:
         params['timeout'] = timeout
     if allowed_updates:
-        params['allowed_updates'] = allowed_updates
+        params['allowed_updates'] = json.dumps(allowed_updates)
     return await _process_request(token, method_name, params=params, request_timeout=request_timeout)
 
 async def _check_result(method_name, result):
